src/docker/functions/container/create-container.ts

```diff
@@ -59,6 +60,12 @@ export const createContainer = async (options: CreateContainerOptions): Promise<
 
 type DockerodeEnvironment = string[];
 
+type Volumes = { [volumes in string]: Record<string, unknown> };
+const getVolumes = (...volumeNames: string[]) => {
```

Comment:
this can just be a `string[]` without varargs to remove the need to spread on line 53